docs/evaluate/development-production-features/multi-tenant.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: multi-tenant
+title: Multi-tenant - Temporal feature
+description: Learn about Temporal Cloud's multi-tenant architecture and how it enhances scalability, efficiency, and cost-effectiveness.
+sidebar_label: Multi-tenant
+tags:
+- multi-tenant
+
+keywords:
+- multi-tenant
+- Temporal Cloud
+- cloud architecture
+- scalability
+- cost-effectiveness
+- noisy neighbor 
+- database performance
+- high throughput
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Temporal Cloud's architecture scales to handle multiple tenants efficiently.
```

Comment:
Seems a bit strange to lead with "Temporal Cloud" here and let people figure out by implication that this is a Cloud feature... I notice the page before this one in the sidebar (Low latency) is like that too. Are we just putting Cloud features side by side with platform features in the Evaluate section?

Reply:
Not neccessarily. Put we are grouping cloud features with self hosted (all of them together) in the features section. Open to any ideas about recategorization as the sidebar gets longer.